### DISCUSSION TITLE
Home tab proposal

### DISCUSSION THREAD

**Discussion by BlishHUD** [blish-hud/Blish-HUD#795] [Module Ideas]
**QuitarHero at 6/26/2020 7:35:57 PM +00:00**

![image](https://user-images.githubusercontent.com/1950594/95031147-4c446d00-0682-11eb-91a8-56a356ff0d0b.png)

Home screen mockup. 

REF: https://discordapp.com/channels/531175899588984842/536970543736291346/726158836171407371

**Comment by entrhopi** [comment]
I would suggest creating a 'Home Tab' module for Blish and then creating a setting in Blish itself where the user can choose a module that should be opened by default on launch (with Blish Settings
being the fallback)
